.github/workflows/test.yml

```diff
@@ -0,0 +1,56 @@
+---
+## Test with multiple ruby images and frameworks
+name: test
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - v*.*.*
+  pull_request:
+    branches:
+      - main
+
+permissions:
+  contents: read
+
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby-image:
+          - 'ruby:2.7'
```

Comment:
I think we should add ruby:3.2 and 3.1
We can probably remove ruby:2.3

Reply:
https://github.com/elastic/ecs-logging-ruby/pull/28/commits/5e0d6a6e03f11863d54a119735a3a0ebf34a204a